### ISSUE TITLE
Multiple word query

### ISSUE THREAD

**Issue by lmburns** [alx741/vinfo#6]
I'm unsure if this works for other packages; however, with `zsh` and the package `zsh-doc` from the AUR, queries such as `info zsh echo` are valid and open up the documentation where `echo` is described. This is sort of the main reason I use the `info` command to begin with.

Though, when using `Vinfo`, only a single word is allowed. I'm not very competent in Vimscript to really know how to modify this to allow for a query like the one I mentioned above. I tried using quotes around the query and that didn't seem to work. So, I'm creating this asking for a feature request of a multi-worded query

Editing to say that commands like the following all work on the command line:
```zsh
$ info zsh 'Parameter Expansion'
$ info zsh Parameter\ Expansion
```

I've tried similar ones with `Vinfo`. Perhaps I'm doing something wrong if it is already allowed

**Comment by alx741** [comment]
Yeah I don't think that would work right now, will try and make it happen
